### PR TITLE
#197 on_init

### PR DESCRIPTION
**Pull request by FredLL-Avaiga**
- em => strong
- npm version 1.0.1
- clean up

### CODE REVIEW

**Review by dinhlongviolin1 — APPROVED** [review]
Looks good to me

File: taipy/gui/gui.py [hunk 1]
```diff
@@ -82,6 +82,9 @@ class Gui:
             It defaults to the _on_change_ global function defined in the Python
             application. If there is no such function, user interactions will not trigger
             anything.
+        on_init (Callable): The function that is called on the first connection of a new user.<br/>
+            It defaults to the _on_init_ global function defined in the Python
```
Comment (FabienLelaquais) [inline taipy/gui/gui.py]: <pre>
_on_init_
</pre>
->
<pre>
`on_init()`
</pre>

**Review by FabienLelaquais — APPROVED** [review]
"That's one small step for a developer, one giant leap for users"

File: taipy/gui/gui.py (hunk 1 above)
Comment (FredLL-Avaiga) [reply taipy/gui/gui.py]: 👍

**Pull request merged**